test/strategies/mintsongs-get-tokenuri/transformer_test.mjs

```diff
@@ -0,0 +1,88 @@
+// @format
+import test from "ava";
+import Ajv from "ajv";
+import addFormats from "ajv-formats";
+import { track } from "@neume-network/schema";
+
+import { onLine } from "../../../src/strategies/mintsongs-get-tokenuri/transformer.mjs";
+
+const ajv = new Ajv();
+addFormats(ajv);
+
+test("mintsongs-get-tokenuri transformer", async (t) => {
+  const payload = {
+    metadata: {
+      block: { number: "15249235" },
```

Comment:
if you pass block.number in to the transformer and don't parse it as an integer, neume-network/schema will throw errors later.